metasv/generate_sv_intervals.py

```diff
@@ -93,22 +147,285 @@ def generate_sc_intervals_callback(result, result_list):
     if result is not None:
         result_list.append(result)
 
+def infer_svtype(aln, isize_mean, isize_sd, num_sd=2):
+    min_isize = isize_mean - num_sd * isize_sd
+    max_isize = isize_mean + num_sd * isize_sd
+    if aln.mate_is_unmapped:
+        return "INS"
+    if aln.tid != aln.rnext:
+        return "CTX,INS"
```

Comment:
Let me now when you are done with reviewing and I will run a small test with these changes and we will be good to merge.